src/main/java/seedu/address/storage/AddressBookStorage.java

```diff
@@ -5,10 +5,11 @@
 import java.util.Optional;
 
 import seedu.address.commons.exceptions.DataConversionException;
-import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.PersonBook;
+import seedu.address.model.ReadOnlyPersonBook;
 
 /**
- * Represents a storage for {@link seedu.address.model.AddressBook}.
+ * Represents a storage for {@link PersonBook}.
  */
 public interface AddressBookStorage {
 
```

Comment:
Should this be named differently since it only deals with PersonBook storage?

Reply:
Will be resolved in a future PR when PropertyBookStorage is implemented as well, as this will take time to refactor as well